### PR TITLE
[FEATURE] Reflect specVersion: 4.0 into documentation

### PR DESCRIPTION
**Pull request by d3xter666**
JIRA: CPOUI5FOUNDATION-835

https://github.com/SAP/ui5-tooling/issues/701

### CODE REVIEW

File: docs/pages/Configuration.md [hunk 1]
```diff
@@ -750,6 +751,26 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 4.0
+
+**Breaking changes:**
+
+- @ui5/builder: Remove bundle option [`usePredefineCalls`](#properties) 
+- @ui5/builder: ui5-builder: New option “async“ for [builder.bundles.bundleDefinition.section](#properties)
+- @ui5/builder: Task API "namespace" option got renamed to ["projectNamespace"](https://sap.github.io/ui5-tooling/v4/api/module-@ui5_builder_tasks_bundlers_generateStandaloneAppBundle.html)
```
Comment (RandomByte) [inline docs/pages/Configuration.md:760]: This is supposed to only list changes in the Specification Version (i.e. the schema or functionality that explicitly checks the spec version).

API changes are usually independent from the Specification Version. For example this one affects projects of all Specification Versions and should therefore be mentioned in the UI5 Tooling 4.0 changelog but not here in the Specificatgion Version 4.0. Same for the rest of the chapter. See spec version 3.2 for an example.

**Review by RandomByte — CHANGES_REQUESTED** [review]

File: docs/pages/Configuration.md (hunk 1 above)
Comment (d3xter666) [reply docs/pages/Configuration.md:760]: Thanks!

File: docs/pages/Configuration.md [hunk 2]
```diff
@@ -750,6 +751,15 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 4.0
+
+**Breaking changes:**
+
+- Remove bundle option [`usePredefineCalls`](#properties)
```
Comment (RandomByte) [inline docs/pages/Configuration.md:758]:
```suggestion
- Remove bundle option [`usePredefineCalls`](#properties). UI5 CLI v4.0.0 and above will always use predefine calls in bundles, making this option obsolete.
```

File: docs/pages/Configuration.md [hunk 3]
```diff
@@ -750,6 +751,15 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 4.0
+
+**Breaking changes:**
+
+- Remove bundle option [`usePredefineCalls`](#properties) 
+- New option “async“ for [builder.bundles.bundleDefinition.section](#properties)
```
Comment (RandomByte) [inline docs/pages/Configuration.md:759]: The async flag is not yet documented at the linked chapter. Should we do that in this PR or separately?

**Review by RandomByte — CHANGES_REQUESTED** [review]

File: docs/pages/Configuration.md (hunk 2 above)
Comment (RandomByte) [reply docs/pages/Configuration.md:758]: Just to give some context on why it has been removed and what the behvavior looks like now.

File: docs/pages/Configuration.md (hunk 3 above)
Comment (d3xter666) [reply docs/pages/Configuration.md:759]: Thanks!
I'll add it with this PR

Comment (d3xter666) [reply docs/pages/Configuration.md:760]: Done

**Review by RandomByte — APPROVED** [review]

**Pull request merged**